### PR TITLE
More understandable onboarding process

### PR DESCRIPTION
**Pull request by sandakersmann**
This will make the onboarding process more understandable for newbies.

### CODE REVIEW

**Comment by sandakersmann** [comment]
@willyfromtheblock What do you think?

**Comment by sandakersmann** [comment]
@willyfromtheblock Will this be included in v0.8.7 ?

**Comment by willyfromtheblock** [comment]
I consider your suggestions to be too wordy. 
They don't improve the experience but throw more text at the user, which is not something you want.

**Comment by sandakersmann** [comment]
I disagree. I think a newbie that does not know what's going on would appreciate a more clear explanation. People I have onboarded actually asked for this.

File: assets/translations/en.json [hunk 1]
```diff
@@ -214,7 +214,7 @@
     "server_settings_alert_hardcoded_content": "Hard coded servers can not be deleted, only deactivated.",
     "server_settings_error_no_server_left": "At least one connectable server is required!",
     "setup_continue_alert_title": "Before you continue",
-    "setup_continue_alert_content": "Make sure you have saved your seed phrase.\nYou can access it later on in the app settings.",
+    "setup_continue_alert_content": "Make sure to save your seed phrase before receiving substantial funds.\nYou can access it later on in the app settings.",
```
Comment (willyfromtheblock) [inline assets/translations/en.json:217]: We actively discourage having "substantial funds" in this app anyway.

File: assets/translations/en.json [hunk 2]
```diff
@@ -227,14 +227,14 @@
...
-    "setup_seed_slider_label": "Slide to change the number of words",
+    "setup_seed_slider_label": "Slide to change the number of words (no need to change)",
     "setup_title": "Welcome",
     "setup_subtitle": "to Peercoin Wallet",
     "setup_text1": "If you have already created a wallet and you still have your seed phrase, then choose",
     "setup_text2": "create your new Peercoin wallet",
     "setup_text3": "or",
     "setup_save_title": "Create Wallet",
-    "setup_save_text1": "The following word list is called Seed. It is a complex password that will give full access to your wallet and to all your coins.",
+    "setup_save_text1": "The following word list is called a seed. It is a complex password that will give full access to your wallet and to all your coins. Do not take a screenshot. It is safest to press export and abort the process. Just write it down on a piece of paper as a backup. You can find the seed later in the app settings.",
```
Comment (willyfromtheblock) [inline assets/translations/en.json:237]: That amount of text won't fit the current layout. 
Why abort the process? You can always access the seed later. You never want to break user flow in a way like this.

Comment (sandakersmann) [reply assets/translations/en.json:237]: There is no button for "continue" before you press "export"

Comment (willyfromtheblock) [reply assets/translations/en.json:237]: Yes, people are forced to export by design in the user flow. Why break it or actively discourage them to do something the flow forces them to?

Comment (sandakersmann) [reply assets/translations/en.json:237]: Exporting seed phrases is not safe for the average user. People will past it in any number of unencrypted cloud storage solutions. Many people get hacked this way.
